Symfony/CS/Fixer/Symfony/MethodSeparationFixer.php

```diff
@@ -156,6 +159,42 @@ private function fixSpaceAboveMethod(Tokens $tokens, $classStart, $methodIndex)
             }
         }
 
+        // deal with comments above a method
+        if ($tokens[$nonWhiteAbove]->isGivenKind(T_COMMENT)) {
+            if ($firstMethodAttrIndex - $nonWhiteAbove === 1) {
+                // no white space found between comment and method start
+                $this->correctLineBreaks($tokens, $nonWhiteAbove, $firstMethodAttrIndex, 1);
+
+                return;
+            }
+
+            // $tokens[$nonWhiteAbove+1] is always a white space token here
+            $lineBreakCount = substr_count($tokens[$nonWhiteAbove + 1]->getContent(), "\n");
+            if ($lineBreakCount > 1) {
+                // more than one line break, always bring it back to 2 line breaks between the method start and what is above it
+                $this->correctLineBreaks($tokens, $nonWhiteAbove, $firstMethodAttrIndex, 2);
+
+                return;
```

Comment:
>I guess what you're saying is that you prefer the 2nd option, that's all.

Not really, because I don't care much about it and I'm happy to refactor to whatever is needed.
I'm trying to say that while discussing what is a preference, I think we can do without qualifications as 'terribly wrong', since without arguments it doesn't add anything useful IMO.

I hope someone who knows what the code should like is willing to take over this PR, because I'm lost here/